backend/api/prisonApi.ts

```diff
@@ -323,7 +323,7 @@ export const prisonApiFactory = (client) => {
     get(context, `/api/bookings/${bookingId}/caseNotes/NEG/IEP_WARN/count?fromDate=${fromDate}&toDate=${toDate}`)
 
   const getPrisonerVisitBalances = (context, offenderNo) =>
-    get(context, `/api/bookings/offenderNo/${offenderNo}/visit/balances`)
+    get(context, `/api/bookings/offenderNo/${offenderNo}/visit/balances?allowNoContent=true`)
```

Comment:
new parameter in prison api that returns no content (204) rather than not found (404) if there are no visit balances set